sklearn/preprocessing/_data.py

```diff
@@ -649,6 +651,7 @@ def minmax_scale(X, feature_range=(0, 1), *, axis=0, copy=True):
     X = check_array(
         X, copy=False, ensure_2d=False, dtype=FLOAT_DTYPES, force_all_finite="allow-nan"
     )
+
```

Comment:
Can you remove the added newline here, in order to avoid adding unrelated change?